packages/domain/src/infrastructure/starknet/uri.rs

```diff
@@ -81,7 +81,7 @@ impl StarknetModel<Erc3525Metadata> for UriModel<Erc3525> {
     async fn load(&self) -> Result<Erc3525Metadata, ModelError> {
         let metadata: Erc3525Metadata = self
             .client
-            .get(self.ipfs_link.clone())
+            .get(self.ipfs_link.to_owned())
```

Comment:
IPFS? What about SVG data URIs?

Reply:
for the moment it fetches a url as https `ipfs_link` var name is not that much related rn. It will be handled easily since contract will directly return a json afterwards